src/module.ts

```diff
@@ -135,7 +142,11 @@ export default defineNuxtModule<ModuleOptions>({
     options.api.methods = moduleOptions.api.methods.length > 0 ? moduleOptions.api.methods : ['patch', 'delete', 'get', 'post']
     nuxt.options.runtimeConfig.session = defu(nuxt.options.runtimeConfig.session, options)
     nuxt.options.runtimeConfig.public = defu(nuxt.options.runtimeConfig.public, { session: { api: options.api } })
-
+    // setup unstorage
+    nuxt.options.nitro.virtual = defu(nuxt.options.nitro.virtual,
+      {
+        '#session-driver': `export { default } from '${builtinDrivers[options.session.storageOptions.driver]}'`
+      })
```

Comment:
Can you link any docs about this option? I'm trying to figure out how to properly type this kind of virtual module and get rid of the `@ts-ignore`

Reply:
I've had a really hard time finding any real documentation. Someone in the nuxt discord pointed out that this was being done in the Nuxt Content module [here](https://github.com/nuxt/content/blob/bfadb27cf281c1098fec0ae0d89bf680a1aa83ef/src/runtime/server/storage.ts#L18) and they used @ts-ignore there too.

All nitro provides is this https://nitro.unjs.io/config#virtual but in their source code for this is here https://github.dev/unjs/nitro/blob/5da951b1242fc3eeffa735bdd3595849d4365888/src/rollup/plugins/virtual.ts#L4 which looks like its based off of rollup' virtual modules https://www.npmjs.com/package/@rollup/plugin-virtual

Reply:
Thanks for digging deep into it! I can also see that [they've also `ts-ignored` `#imports`](https://github.com/nuxt/content/blob/bfadb27cf281c1098fec0ae0d89bf680a1aa83ef/src/runtime/server/storage.ts#L15-L16), while [we don't](https://github.com/sidebase/nuxt-session/blob/c0cfeb729dc369bfea2b461172dafc1084ddc85b/src/runtime/server/middleware/session/sessionStorage.ts#L4). Any insights on this one, @BracketJohn?

Otherwise, I would say it's fine to ignore it in this PR and figure it types later.

Reply:
In light of nuxt doing this in one of theor modules, I think it Would be ok to leave this for now, so that we can move towards PR completion + release (: We can open an issue to improve this later. 

I think I saw a solution to this in the supabase nuxt module and _may_ habe solved this in nuxt-auth by generating a types file (but am very uncertain if this is the same thing needed here - let's rather aim to merge than perfect this part).